zimui/src/pages/HomePage.vue

```diff
@@ -17,23 +17,41 @@ watch(params, () => {
   }
 })
 
+const errMessage = computed(() => main.errorMessage)
+
 // fetch channel data and set default topic if needed
 onMounted(async () => {
-  await main.fetchChannel()
-  if (topic.value === undefined && main.channelData != null) {
-    topic.value = main.channelData.rootSlug
+  try {
+    await main.fetchChannel()
+    if (topic.value === undefined && main.channelData != null) {
+      topic.value = main.channelData.rootSlug
+    }
+  } catch (error) {
+    console.error('Error fetching channel data:', error)
+    // Handle error here, for example, show an error message to the user
   }
 })
 
 import TopicHome from '../components/TopicHome.vue'
 </script>
 
 <template>
-  <div class="d-flex flex-column h-100">
+  <div>
```

Comment:
Again, I don't get why you need these extra divs + new `d-flex flex-column h-100` classes

Reply:
- removed the extra div
- `d-flex flex-column h-100` was already in the original code

Reply:
Still one extra `div` which is useless AFAIK

Reply:
You didn't needed this extra div in `TopicHome.vue` for instance